pkg/reconcilermanager/controllers/util.go

```diff
@@ -205,30 +206,52 @@ const (
 	helmSyncPassword = "HELM_SYNC_PASSWORD"
 )
 
-// helmSyncEnvs returns the environment variables for the oci-sync container.
-func helmSyncEnvs(repo, chart, version, releaseName, namespace string, auth configsync.AuthType, period float64) []corev1.EnvVar {
+// helmSyncEnvs returns the environment variables for the helm-sync container.
+func helmSyncEnvs(helm *v1beta1.Helm) []corev1.EnvVar {
 	var result []corev1.EnvVar
+	helmValues := ""
+	helmValuesFiles := ""
+	if len(helm.ValuesFiles) > 0 {
+		helmValuesFiles = strings.Join(helm.ValuesFiles, ",")
+	}
+	if helm.Values != nil {
+		var vals []string
+		for key, val := range helm.Values.Object {
+			vals = append(vals, fmt.Sprintf("%s=%v", key, val))
+		}
+		helmValues = strings.Join(vals, ",")
```

Comment:
nit: there will be a trailing "," (i.e. `k1=v1,k2=v2,`) - would that cause any issues?

Reply:
Per discussion, `Join` won't add trailing `,`